api/src/services/streams-service.ts

```diff
@@ -127,7 +127,7 @@ export class StreamsService {
 								return channelData;
 							} catch (e) {
 								this.logger.error('could not parse maskedPayload');
-								return;
+								// return;
 							}
 						})
 						.filter((c: ChannelData | undefined) => c);
```

Comment:
`.map()` and `.filter` also support optional chaining like `?.map()` and `?.filter`
This would prevent an exception in case where none of the `maskedPayload ` can be parsed

Reply:
I've tried this but it did not return [2,4,5] instead an error occured:
see: https://www.typescriptlang.org/play?#code/MYewdgzgLgBAnjAvDA2gJgDQGYMBYMCsAugPwB0AtgIYAOAFAB6IB8ABQN4wCWAZo4oiwBKGOygALAE4gA7jDABTOQFFJ0yXQDkDHQ01CAvjEkKoAV0lgYDANwGh5HlwA2UBRqbMGQgFChIIM4KZM4gAOZ0cEJAA

```
const y = [2,3,4,5]?.map(x=> { if(x==3) {throw new Error('xxxx')} return x;})?.filter(x=>x)
console.log(y)
```

Reply:
sure, this is because you throw an error :) 

If you modify it like `const y = [2,3,4,5]?.map(x=> { if(x==3) {return null;} return x;})?.filter(x=>x)` it will return `[2,4,5]`